src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -157,6 +158,17 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
                     this.Fixes.Add(fixModel);
                 }
             }
+
+            if (result.PropertyNames?.Any() == true)
```

Comment:
>== true [](http://example.com/codeflow?start=44&length=7)

do we need == ture here

Reply:
its because result.PropertyNames? can be null so need compare it with true